src/node-red/terminology/index.md

```diff
@@ -0,0 +1,92 @@
+---
+eleventyNavigation:
+  key: Terminology
+title: Node-RED Terminology
+description: Explore key terms and concepts used within the Node-RED community, including flows, nodes, subflows, workspace, and more.
+tags:
+    - node-red
+    - node-red terminology
+    - node-red concepts
+    - node-red key terms
```

Comment:
```suggestion
meta:
   title: Node-RED Terminology
   description: Explore key terms and concepts used within the Node-RED community, including flows, nodes, subflows, workspace, and more.
   keywords: node-red, node-red terminology, node-red concepts, node-red key terms
```

Hi @sumitshinde-84, the title is being rendered in a section that we don't need for this. This one is on me, I should change the template, I tested a couple just now but none seems like the right fit. I'll take care of that in a different PR. However, we can still have a title without rendering it, if we add it under the `meta` key. 

Regarding the tags, Eleventy uses them to build posts collections, so this would be added to `/blog/node-red/`, which we don't want. So for the content that is not a blog article, let's add `meta.keywords` instead. [Here's a short explanation](https://flowfuse.com/handbook/customer/marketing/website/#meta-keywords) on how this works.

Reply:
Thanks @Yndira-E for the update